docs/pages/accounts/working-together.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Working Together
+---
+
+You can add other Expo users or members to your Personal Account. Members have access to all of the projects belonging to the account. The type of access depends on the given role.
+
+## Adding Members
+
+You can invite new members to your Personal Account, or any account you administrate, from the members page under Settings at [https://expo.io/dashboard/ORGANIZATION/settings/members](https://expo.io/dashboard/ORGANIZATION/settings/members). You can only add existing Expo users as a member, you can direct them to [https://expo.io/signup](https://expo.io/signup) if they don't have an account yet.
+
+> When adding new developers to your projects, who are publishing updates or create new builds, make sure to add the [`owner`](../../versions/latest/config/app/#owner) property to your project app manifest.
+
+## Managing Access
+
+Access for members is managed through a role-based system. Users can have the _owner_, _admin_, _developer_, or _viewer_ role within Personal Accounts.
+
+| Role          | Description                                                                                                                                           |
+| ------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Owner**     | Can take any action on the account or on any projects, even deleting them.                                                                            |
```

Comment:
The owner permission here and on line 15 should be moved to the Orgs docs changes.  Owner only applies to Organization accounts

Reply:
Good point, I'll strip it from this PR!